packages/tools/addon/components/cs-field-editor.js

```diff
@@ -13,6 +13,13 @@ export default Component.extend({
   enabled: true,
   errors: null,
 
+  init() {
+    this._super(...arguments);
+    if (!this.onchange) {
+      this.onchange = () => {};
+    }
```

Comment:
what's the advantage of this over just defining `onchange() {}` in the class itself?

Reply:
Nothing, at one point I had a bug that I tried to trace down and forgot to refactor. Thanks!